client/state/reader-ui/reducer.js

```diff
@@ -6,6 +6,32 @@ import sidebar from './sidebar/reducer';
 import { combineReducers, withStorageKey } from 'state/utils';
 import cardExpansions from './card-expansions/reducer';
 import hasUnseenPosts from './seen-posts/reducer';
+import { DESERIALIZE, SERIALIZE } from 'state/action-types';
+
+/**
+ * Keep the last reader stream path selected by the user, for the purpose of autoselecting it
+ * when user navigates back to Reader
+ *
+ * @param state redux state
+ * @param action redux action
+ *
+ * @returns {null|string} last path selected
+ */
+export const lastPath = ( state = null, action ) => {
+	switch ( action.type ) {
+		case SERIALIZE:
+		case DESERIALIZE:
+			return state;
```

Comment:
If you have multiple tabs open against the reader, they may conflict over control of this state. To test:

1. open the reader in two tabs
2. go to feed A in tab A, feed B in tab B
3. leave the reader in both
4. visit `/read` in both. Both will end up on the last feed seen in the last browser to visit a feed page

Is that the expected behavior?

Reply:
> visit /read in both. Both will end up on the last feed seen in the last browser to visit a feed page

I think so 🤔, hard to solve, what would be ideal?